blog/_posts/2016-09-28-rubycats-at-codemotion-warsaw-and-euruko.md

```diff
@@ -0,0 +1,152 @@
+---
+title: RubyCats at Codemotion Warsaw and EuRuKo!
+layout: post
+created_at: Thu Sep 29 2016
+permalink: blog/2016-09-29-rubycats-at-codemotion-warsaw-and-euruko
+current: blog
+author: Iza & Kinga
+twitter: RubyCatsTeam
+categories: student-posts
+---
+
+## <span class="color-red">Codemotion Warsaw 2016 Conference</span>
+
+On 15-16 September we had a chance to participate in an awesome event – Codemotion Warsaw 2016 Conference. Many interesting talks and people! But let’s start from the beginning.
+
+We started with keynote by James Weaver about "Navigating all the knowledge". It was fascinating to see how we can look for what particular records, e.g. people, genes etc., have in common. One of exercise case was what Michał Pazdan and Robert Lewandowski have in common ;)
+
+![RubyCats](/img/blog/2016/rubycats-codemo-weaver.jpg)<font color="grey"><small><i>James Weaver</i></small></font>
```

Comment:
Hi @alicetragedy I'm very sorry for late reply, I spent almost the whole flu-day in bed and getting back to life and work. Thank you for your feedback :)
Sorry for a typo, of course I'll correct it.
All images were taken by us instead of the one from our LT, which was taken by Team Joda. Should we add the credit to every photo then?
As it's quite late, should I change both dates /filename and permalink/ to 30 Sept?